common/test/java/com/couchbase/lite/internal/core/C4BaseTest.java

```diff
@@ -163,17 +163,13 @@ protected void checkChanges(
     }
 
     protected int getTestDbFlags() {
-        int flags = C4Database.DB_FLAGS;
-        if (C4Database.VERSION_VECTORS_ENABLED) { flags |= C4Constants.DatabaseFlags.FAKE_CLOCK; }
-        return flags;
+        return C4Database.DB_FLAGS | C4Constants.DatabaseFlags.FAKE_CLOCK;
```

Comment:
The tests should all pass without this flag present.

Reply:
They do!   Cool!
Removed.